src/actions/notificationActions.js

```diff
@@ -1,8 +1,12 @@
 export const SHOW_NOTIFICATION = 'AOR/SHOW_NOTIFICATION';
 
-export const showNotification = (text, type = 'info') => ({
+export const showNotification = (
+    text,
+    type = 'info',
+    autoHideDuration = undefined
```

Comment:
a default value of `undefined` makes no sense. Please remove the default value.

Reply:
it's to explicitly show that is should be undefined

Reply:
removed then